inc/includes.php

```diff
@@ -91,23 +91,16 @@
 }
 
 /* On startup, register all plugins configured for use. */
-if (!isset($PLUGINS_INCLUDED)) {
```

Comment:
it should be not used a lot currently but this serves to bypass plugin loading in some dev (found a mention in formcreator unit tests)